### PR TITLE
feat: add Appium integration tests to workflow

### PR DESCRIPTION
**Pull request by ssmathistad**
- add Appium integration tests for iOS/Android to mobile pipeline workflow.

### CODE REVIEW

**Review by MisterC500 — APPROVED** [review]

File: appium/ios/appium-ios-tests.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+	path: '/wd/hub',
+	port: 4723,
+	connectionRetryTimeout: 900000,
+	wdaLaunchTimeout: 900000,
+	capabilities: {
+        platformName: "iOS",
+        platformVersion: "15.0",
+        deviceName: "iPhone 11",		
+        automationName: "XCUITest",
+		app: "/Users/runner/work/mobile-pipeline-poc/mobile-pipeline-poc/ReactNativeSemaphoreNew.xcarchive/Products/Applications/ReactNativeSemaphoreNew.app",
```
Comment (MisterC500) [inline appium/ios/appium-ios-tests.js:14]: Could we fix the indentation of these lines?

File: sleep.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (MisterC500) [inline sleep.sh:1]: Are we still using the sleep script?

File: appium/ios/appium-ios-tests.js (hunk 1 above)
Comment (ssmathistad) [reply appium/ios/appium-ios-tests.js:14]: It is properly formatted in the repo [link](https://github.com/liatrio/mobile-pipeline-poc/blob/appium-workflow-integration/appium/ios/appium-ios-tests.js) but not showing up in the pr.

File: sleep.sh (hunk 2 above)
Comment (ssmathistad) [reply sleep.sh]: Removed

**Pull request merged**